apps/condo/domains/banking/utils/validate/countrySpecificValidators/ru.validator.js

```diff
@@ -37,17 +40,17 @@ const validateRuNumber = (number, routingNumber, errors) => {
 /**
  * For RU routing number:
  * *** In Russia, number routing is equivalent to BIC***
- * 1) For RU organizations country code (first two digits) is 04
+ * 1) For RU organizations country code (first two digits) is 04 or 01 (for government organizations)
  *
  * @param {string} routingNumber
  * @param {[]} errors
  */
 const validateRuRoutingNumber = (routingNumber, errors) => {
-    const WRONG_RU_COUNTRY_CODE = 'For RU organizations country code is 04, but routing number have '
+    const WRONG_RU_COUNTRY_CODE = 'For RU organizations country code is 04 or 01, but routing number has '
```

Comment:
Seems that this sentence ends abruptly

Reply:
it uses `routingNumber` variable at the end (to compare what is expected vs what is actually there)